### ISSUE TITLE
Flag to check for internal compiler errors

### ISSUE THREAD

**Issue by fenhl** [rust-lang/cargo-bisect-rustc#34]
It would be nice as a convenience if this could provide an `--ice` flag to check specifically for internal compiler errors rather than any error.

**Comment by pnkfelix** [comment]
We definitely need this.

Specifically, such a flag should treat a compiler error as a "pass" and a ICE as a fail.

(we might need a separate mode that treats a compiler error as a "pass" and any other behavior (compiler acceptance or an ICE) as a fail. I'm not sure whether such work needs to be done up front, nor whether it would make sense to try to fold it into this task.)

**Comment by chrissimpkins** [comment]
#47 adds a shell script that can be used to automate this until the functionality is implemented in the tool

**Comment by pnkfelix** [comment]
Hmm. I've been working on putting support into `cargo-bisect-rustc` itself for this and a few other testing variations.

I've been assuming that building the support into `cargo-bisect-rustc` would be best for the long term, but maybe that is an incorrect assumption.

**Comment by pnkfelix** [comment]
(the main reason I haven't put up a PR yet is due to the issue described by PR #45 )

**Comment by chrissimpkins** [comment]
> Hmm. I've been working on putting support into cargo-bisect-rustc itself for this and a few other testing variations.
>
>I've been assuming that building the support into cargo-bisect-rustc would be best for the long term, but maybe that is an incorrect assumption.

Definitely no intent to derail that effort.  These were simple one-liner shell scripts and I figured that they might be of use to others until something is implemented in the tool.  I used the ICE stderr substring script in a few cargo bisect reports over the weekend - as an aside very much under the assumption that this is helpful / desirable from a new member of the "ICE-breaker Cleanup Crew" team :) - and it works well for now.

I spoke with Santiago about builtin subcommand functionality on Zulip today.  If that is something that you intend to do, I am definitely interested to lend a hand if there is anything that I can help with the effort.  I am relatively new to Rust and it might require a bit of mentorship at this stage.  But I have time and an interest if that helps to push it forward.
